yolov8_ros/yolov8_ros/yolov8_ros/yolov8_node.py

```diff
@@ -278,9 +328,12 @@ def image_cb(self, msg: Image) -> None:
 
                 detections_msg.detections.append(aux_msg)
```

Comment:
Can this go?

Reply:
Keeping old publisher for visualization purposes. Reverted to commit 744ad748b19208b1077d615650d5b2944bdaa700.